app/models/payment_request.rb

```diff
@@ -24,6 +24,11 @@ class PaymentRequest < ApplicationRecord
   def invoice_ids
     applied_invoices.pluck(:invoice_id)
   end
+
+  def increment_payment_attempts!
+    increment(:payment_attempts)
+    save!
```

Comment:
why not directly?
```suggestion
    increment!(:payment_attempts)
```

Reply:
rubocop does not allow `increment!` because it skips ActiveRecord validations, callbacks, etc.